spec/hiera.yaml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
Copy of hiera.yaml to spec/hiera.yaml PLUS spec_tests.yaml in hierarchy for testing